test/LoggerAbstractServiceFactoryTest.php

```diff
@@ -291,7 +291,7 @@ public function testWillInjectWriterPluginManagerIfAvailable()
     public function testWillInjectProcessorPluginManagerIfAvailable()
     {
         $processors = new ProcessorPluginManager(new ServiceManager());
-        $mockProcessor = $this->getMock('Zend\Log\Processor\ProcessorInterface');
+        $mockProcessor = $this->getMockBuilder('Zend\Log\Processor\ProcessorInterface')->getMock();
```

Comment:
I previously created another pr for ziactoros and @weierophinney warn me about using `getMockBuilder()` instead of `createMock` to work with PHPUnit 4 and 5.